src/patternfly/base/_common.scss

```diff
@@ -1,26 +1,4 @@
 // Set common reset styles for patternfly components
-// remove in breaking change. this should come from globals, and can be opted out
-[class*="pf-c-"] {
-  &,
-  &::before,
-  &::after {
-    padding: 0;
-    margin: 0;
-    background-color: transparent;
-  }
-}
-
-// remove in breaking change
-@if $pf-global--unset-root-font-size {
```

Comment:
we can also remove this https://github.com/patternfly/patternfly/blob/14e45911682ee81e822cecc02c22cb380a98c05b/src/patternfly/sass-utilities/scss-variables.scss#L15